src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -13,7 +13,7 @@
 namespace Microsoft.ML.Runtime.Internal.Utilities
 {
     /// <summary>
-    /// This class takes care of downloading resources needed by TLC components. Resources are located in
+    /// This class takes care of downloading resources needed by ML.Net components. Resources are located in
```

Comment:
>Net [](start = 68, length = 3)

It is I think branded as NET not Net.

Reply:
Thank you! I had that wrong, just corrected it in another commit